unit_tests.sh

```diff
@@ -93,33 +93,29 @@ for group in $test_groups ; do
 
 			run_test 1 $0:$LINENO "unknown TEST" TEST=invalid TEST_PKG=valid
 
-			run_test 1 $0:$LINENO "empty catkin workspace" TEST_PKG=valid 'BEFORE_SCRIPT="rm valid"'
+			# TODO(mlautman): Decide if we should keep this test as an empty ros workspace doesn't
+			# 				  seem to be an acutual issue
+			# run_test 1 $0:$LINENO "empty ROS workspace" TEST_PKG=valid 'BEFORE_SCRIPT="rm valid"'
 
 			;;
 		warnings)
 			run_test 0 $0:$LINENO "'warnings' package with warnings allowed" TEST_PKG=warnings WARNINGS_OK=true
 			run_test 1 $0:$LINENO "'warnings' package with warnings forbidden" TEST_PKG=warnings WARNINGS_OK=false
 			run_test 0 $0:$LINENO "'valid' package with warnings forbidden" TEST_PKG=valid WARNINGS_OK=false
 			;;
-		catkin_lint)
```

Comment:
This should be commented along with a TODO to implement it with ament lint

Reply:
done